libevm/VMFactory.cpp

```diff
@@ -19,8 +19,8 @@
 #include "EVMC.h"
 #include "SmartVM.h"
 #include "VM.h"
-#include <libdevcore/Assertions.h>
 #include <evmjit.h>
+#include <hera.h>
```

Comment:
what if (HERA) flag not defined?

Reply:
It's always defined.

Reply:
The header is always available, because it's in the submodule.